### PR TITLE
Pythia (Test PR)

### PR DESCRIPTION
**Pull request by leandrogg**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] I have added thorough documentation for my code.
- [ ] I have tagged PR with relevant project labels. I acknowledge that a PR without labels may be dismissed.
- [ ] If this PR addresses a bug, I have provided both a screenshot/video of the original bug and the working solution.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+*.json
+*.arrow
 *.pkl
 ckpts*
 .deepspeed_env
```
Comment (leandrogg) [inline .gitignore:5]: Add more specific commit messages for better understanding of the changes made in each PR.

File: configs/train/finetune_pythia.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+lora: false
+warmup_steps: 500
+num_epochs: 3 
+
+# logging
+wandb: true
+wandb_entity: gpt4all
+wandb_project_name: gpt4all
+seed: 42
+
```
Comment (leandrogg) [inline configs/train/finetune_pythia.yaml:33]: The PR diff appears to be a set of configuration settings for training a model. Here are some general suggestions:

* Consider breaking down long lines into multiple lines for better readability.
* Use consistent spacing and indentation throughout the code.
* Use more descriptive variable names instead of generic ones like `lr` and `weight_decay`.
* Consistent use of whitespace between values and their keys would improve readability.
* No syntax errors were found in this PR.

File: data.py [hunk 3]
```diff
@@ -72,7 +72,9 @@ def load_data(config, tokenizer):
         dataset = load_dataset("json", data_files=files, split="train")
 
     else:
-        dataset = load_dataset(dataset_path, split="train")
+        dataset = load_dataset(dataset_path,
+                               split="train",
+                               revision=config["revision"] if "revision" in config else None)
 
     dataset = dataset.train_test_split(test_size=.05, seed=config["seed"])
```
Comment (leandrogg) [inline data.py:80]: The suggested change to `remove_columns` seems reasonable and should improve readability by reducing noise in the output data. The addition of `"id"` to this list appears to be a minor correction that ensures consistency in removing unnecessary columns.

In terms of performance, there are no obvious suggestions for improvement in this diff block.
